xbmc/video/VideoDatabase.cpp

```diff
@@ -4470,10 +4470,10 @@ bool CVideoDatabase::GetVideoSettings(int idFile, CVideoSettings &settings)
       settings.m_CenterMixLevel = m_pDS->fv("CenterMixLevel").get_asInt();
       m_pDS->close();
 
-      if (settings.m_ToneMapParam == 0.0)
+      if (settings.m_ToneMapParam < 0.1f || settings.m_ToneMapParam > 5.0f)
       {
-        settings.m_ToneMapMethod = VS_TONEMAPMETHOD_REINHARD;
-        settings.m_ToneMapParam = 1.0;
+        settings.m_ToneMapMethod = VS_TONEMAPMETHOD_HABLE;
```

Comment:
can you implicitly include cores/VideoSettings.h as a header please. theres a forward declaration in the header, but there should be an implicit inclusion in the implementation file.

Reply:
I'm not sure what this code is needed for. I think it can be removed.

In the tests that I have done adding and removing media library items it has never entered to this block.

It seems that it was a stopgap measure to migrate old database or initialize wrong values. Now if no saved data in DB, the default values from `m_defaultVideoSettings` are used.